examples/seq2seq/seq2seq.ipynb

```diff
@@ -6,8 +6,7 @@
     "colab": {
```

Comment:
<p>consider removing all mentions of <code>configs/default.py</code> etc to better reflect the structure we have in the seq2seq example </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/flax/pull/1943/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Done